Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:13
+RUN npm install -g docsify-cli
+
+# https://github.com/docsifyjs/docsify-cli/issues/78
+RUN apt-get update \
+	&& apt-get install dos2unix \
+	&& dos2unix /usr/local/lib/node_modules/docsify-cli/bin/docsify
+
+EXPOSE 3005
+USER node
+
+WORKDIR /usr/src/app
```

Comment:
UNIX paths IIRC do have some sanity in their naming. Shouldn't this be places in `/var/something` ?

Reply:
I would disagree actually. Most of the node images use `/usr/src/app` so it is more a convention in node docker documentation: https://github.com/nodejs/docker-node/blob/5351774b970cd32b80fc0c47a8abff5ba155fccc/docs/BestPractices.md

Reply:
`/usr/local/bin` is something _very_ different than `/usr/src/app`. This is, on standard UNIX systems, a directory for programs that a normal user may run. Putting binaries in there that the resulting node image can "run" is therefore 'compliant'.

Note: I don't actually know what the _right_ place is to put the source for the directory, and if  `/usr/src/ ` is a convention I simply don't know about, then by all means, let's keep it there. I know this is pedantic, and I'm not saying this to annoy you, but I don't want to ever have to answer the question "why is it in `/usr/src/app`?"

I'll ask our devops peeps who live and breath UNIX.

Reply:
@SleeplessByte can we continue this conversation on Slack sometime? I think I am missing your concerns.

Reply:
Always! Ping me whenever you like :)

Reply:
I was wondering if you've (@efx and @SleeplessByte) managed to come to a conclusion?

Reply:
@ErikSchierboom @SleeplessByte sorry; I have not had the stamina to approach this. I am blocked on understanding *why* specifying the mount point is a problem and would like to understand the object better before changing or merging. Happy to flesh this out here too.

Reply:
No worries. I'll leave it up to @SleeplessByte to check on this.

Reply:
We met and discussed and decided to put within `/opt/<name-of-project>` per our reading of the [FHS standard](https://refspecs.linuxfoundation.org/FHS_2.3/fhs-2.3.html#OPTADDONAPPLICATIONSOFTWAREPACKAGES).